tools/sdg/topics/core_topics.py

```diff
@@ -48,20 +48,34 @@
 _UNDATA_NL_DESCRIPTIONS_FILE = os.path.join(_TMP_DIR,
                                             'undata_nl_descriptions.csv')
 
-API_ROOT = "https://autopush.api.datacommons.org"
+_UNDATA_ILO_ROOT = "dc/g/Custom_UN"
+_UNDATA_ILO_TOPIC_JSON = '../../../server/config/nl_page/undata_ilo_topic_cache.json'
+_UNDATA_ILO_NON_COUNTRY_VARS = '../../../server/config/nl_page/undata_ilo_non_country_vars.json'
+_UNDATA_ILO_MCF_PATH = os.path.join(_TMP_DIR, 'custom_topics_undata_ilo.mcf')
+_UNDATA_ILO_VARIABLES_FILE = 'undata_ilo_variable_grouping.csv'
+_UNDATA_ILO_SERIES_TOPICS_FILE = 'un_ilo_series_topics.csv'
+_UNDATA_ILO_NL_DESCRIPTIONS_FILE = os.path.join(
+    _TMP_DIR, 'undata_ilo_nl_descriptions.csv')
+
+REMOVE_SVG_PREFIX = "Custom_"
+
+API_ROOT = "https://staging.unsdg.datacommons.org"
 API_PATH_SVG_INFO = API_ROOT + '/v1/bulk/info/variable-group'
 API_PATH_PROP_OUT = API_ROOT + '/v1/bulk/property/values/out'
 
 
-def _svg2t(svg):
+def _svg2t(svg, remove_svg_prefix=""):
   # NOTE: Use small "sdg" to avoid overlap with prior topics.
-  return svg.replace('dc/g/SDG', 'dc/topic/sdg').replace('dc/g/', 'dc/topic/')
+  return svg.replace('dc/g/SDG',
+                     'dc/topic/sdg').replace('dc/g/', 'dc/topic/').replace(
+                         f"/{remove_svg_prefix}", "/")
 
 
-def download_svg_recursive(svgs: List[str], nodes: Dict[str, Dict],
-                           keep_vars: Set[str]):
+def download_svg_recursive(svgs: List[str],
+                           nodes: Dict[str, Dict],
+                           keep_vars: Set[str],
+                           remove_svg_prefix=""):
```

Comment:
I regret not wrapping this whole thing in a class :)

Reply:
After working with this script , maybe we can refactor it to a class and to take in a yaml config file for the various sdg/who/ilo configurations